edb/pgsql/metaschema.py

```diff
@@ -2399,9 +2399,9 @@ def _generate_role_views(schema):
             a.rolname                                   AS name,
             a.rolsuper                                  AS is_superuser,
             a.rolcanlogin                               AS allow_login,
-            a.rolpassword                               AS password
+            (d.description)->>'ph'                      AS password
```

Comment:
Do you really think that compressing/obfuscating this field is justified?

Reply:
Not really.  Security through obscurity does not work.  We have to rely on a) That access to `pg_shdescription` is sufficiently restricted; and b) That SCRAM makes brute-force attacks infeasible.

Reply:
Yes. I mean, rename it to `password_hash` or something :)

Reply:
Ah, OK. Done.

Reply:
Ah, OK. Done.